protobuf/crypto/share.proto

```diff
@@ -4,8 +4,8 @@
  */
 syntax = "proto3";
 
-/*option go_package = "github.com/drand/drand/protobuf/crypto";*/
-option go_package = "share";
+option go_package = "github.com/drand/drand/protobuf/crypto";
```

Comment:
should be protobuf/share?

Reply:
I supposed what this means is nothing uses this file.

the directory is 'protobuf/crypto', and that package is what will cause the generated .go file to end up in the right place

Seems like the right answer is to remove the proto / generated file here entirely though?

Reply:
I don’t know enough about this yet to make that call 😢 but I agree with your assertion